insights/specs_insights_archive.py

```diff
@@ -112,11 +112,15 @@ class InsightsArchiveSpecs(Specs):
     numeric_user_group_name = simple_file("insights_commands/grep_-c_digit_.etc.passwd_.etc.group")
     oc_get_bc = simple_file("insights_commands/oc_get_bc_-o_yaml_--all-namespaces")
     oc_get_dc = simple_file("insights_commands/oc_get_dc_-o_yaml_--all-namespaces")
+    oc_get_egressnetworkpolicy = simple_file("insights_commands/oc get egressnetworkpolicy -o yaml --all-namespaces")
```

Comment:
These simple_files are not correct, please rerun the "pip install -e .[develop] for 3.x", you'll have a "bin/mangle" script that can convert simple_command strings to simple_file strings.